fs-dir-common.go

```diff
@@ -119,10 +120,10 @@ func treeWalk(bucketDir, prefixDir, entryPrefixMatch, marker string, recursive b
 	var markerBase, markerDir string
 	if marker != "" {
 		// Ex: if marker="four/five.txt", markerDir="four/" markerBase="five.txt"
-		markerSplit := strings.SplitN(marker, string(os.PathSeparator), 2)
+		markerSplit := strings.SplitN(marker, "/", 2)
```

Comment:
Is there a reason why `filepath.Dir` and `filepath.Base` aren't used to get `dirname(1)` and `basename(1)` given path?